tests/integration/jinad/test_simple_distributed/Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM python:3.7.6-slim
+
+WORKDIR /
+
+RUN apt-get update && \
+    apt-get install --no-install-recommends -y git \
+                                               curl
+
+COPY tests/integration/test_simple_distributed/requirements.txt /tests/integration/test_simple_distributed/requirements.txt
```

Comment:
I know this is copied from the other example, but I think the in-docker files can be less verbose. e.g. only do `COPY tests/integration/test_simple_distributed/requirements.txt /requirements.txt` and adapt the following `RUN` accordingly